python/covid_web/covid_web/rest.py

```diff
@@ -1,7 +1,7 @@
 from pathlib import Path
 from typing import Dict, List, Tuple
 
-from flask import Flask, abort, jsonify
+from flask import Flask, abort, jsonify, url_for
```

Comment:
```suggestion
from flask import Flask
import flask
```
And then use `flask.abort`, `flask.jsonify`, etc.

Reply:
Changed.